lib/scope.js

```diff
@@ -1003,6 +1003,27 @@ module.exports = {
         await scope.uploadFiles( scope, { handle, set_to });
         await scope.afterStep(scope, { waitForShowIf: true });
 
+      } else if (type == `date`) {
+        let set_to_date = set_to;
+        // If the date is specified in the format "today" or "today+2" or "today-1"
+        // it will need some pre-processing. 
+        if (set_to.indexOf(`today`) !== -1) {
+          // Find out how many days to add/subtract
+          let date_delta = set_to.match(/today([+-]\d*)?/)[1];
```

Comment:
The below may be more appropriate as the current regex allows just '-' or just '+' alone. It would then be defined and yet wouldn't be a valid int.

All to explain why requiring at least one digit is probably useful.

```suggestion
let date_delta = set_to.match(/today([+-]\d+)?/)[1];
```